src/distbin-html/an-activity.ts

```diff
@@ -30,12 +30,21 @@ import { internalUrlRewriter } from "./url-rewriter";
 import { IncomingMessage, ServerResponse } from "http";
 import * as marked from "marked";
 import * as url from "url";
+import * as fs from 'fs';
 
 import { createLogger } from "../logger";
 const logger = createLogger(__filename);
 
 const failedToFetch = Symbol("is this a Link that distbin failed to fetch?");
 
+// Highlighting
+const highlightCss = fs.readFileSync(require.resolve('highlight.js/styles/github.css'), 'utf-8');
+marked.setOptions({
+  highlight: function (code) {
+    return require('highlight.js').highlightAuto(code).value;
```

Comment:
Please get `highlightAuto` function not from `require()` but by importing at top:
```
import { highlightAuto } from "highlight.js"
```

That way any module resolution errors happen at time of file evaluation, and not later only once this `highlight()` optoin/method is called.

Reply:
modification made